server/index.js

```diff
@@ -1,36 +1,18 @@
-dotenv.config();
 import express from 'express';
-import dotenv from 'dotenv';
-import cookieParser from 'cookie-parser';
-
-import { notFound, errorHandler } from './middleware/errorMiddleware.js';
-import connectDB from './database/connection.js';
-import userRoutes from './routes/userRoutes.js'
-import Product from './models/Product.js';
+import dotenv from 'dotenv'
+import routes from './routes/auth.js'
+import db from './database/db.js';
 
-const port = process.env.PORT || 4000;
-
-connectDB();
+dotenv.config();
 const app = express();
+const PORT = process.env.PORT || 4000;
+db();
```

Comment:
Reported this multiple times already, `db()` returns `Promise` which should be handled. I recommend learning about JavaScript Promises, before fixing this

Reply:
ok i will learn about it and ill try to fix it

Reply:
can i do like this
async function startApp(){
try{
await db();
   }
console.log('');
}catch(error){
console.error('',error.message);
}
}
startApp();